tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/rdr_observation_source_concept_id_suppression_test.py

```diff
@@ -99,3 +102,12 @@ def setUp(self):
             load_statements.append(query)
 
         self.load_test_data(load_statements)
+
+    def test(self):
+        """
+        Use the default drop rows test function.
+
+        Validates pre-conditions, test execution and post conditions based on
+        the load statements and the tables_and_counts variable.
+        """
+        self.default_drop_rows_test(self.tables_and_counts)
```

Comment:
I would inline the dictionary here. I also would not set `cls.tables_and_counts` anymore since the property is no longer defined by the base class, presumably this was leftover from the prior approach

Reply:
no.  i left it there on purpose.